liby2/src/include/y2/Y2PluginComponent.h

```diff
@@ -62,7 +62,12 @@ class Y2PluginComponent : public Y2Component
     /**
      * The component level the plugin was started in.
      */
+#ifdef __clang__
+    int level __attribute__((__unused__));
```

Comment:
GCC knows `unused` if it is applied to arguments, but here it will cause a warning.

Warning! Unknown warning "-Wwarn-about-warnings"!

(just joking)